src/main/com/fulcrologic/guardrails/utils.cljc

```diff
@@ -84,7 +84,9 @@
 (let [p! persistent!, t transient]                          ; Note `mapv`-like nil->{} semantics
   (defn filter-vals [pred m] (if (nil? m) {} (p! (reduce-kv (fn [m k v] (if (pred v) m (dissoc! m k))) (t m) m)))))
 
-#?(:clj
+#?(:bb
+   (defn atom? [x] (satisfies? clojure.lang.IAtom x))
```

Comment:
@wilkerlucio I think this isn't needed anymore with bb master.

Reply:
true, just checked, removing change